src/components/screens/HomeScreen.tsx

```diff
@@ -52,8 +54,10 @@ const BorderLine = styled.View`
 `;
 
 export const HomeScreen = () => {
-  const publicKey = TEST_PUBLIC_KEY;
-  const networkDetails = TEST_NETWORK_DETAILS;
+  const { account } = useGetActiveAccount();
+  const { network } = useAuthenticationStore();
+  const publicKey = account?.publicKey;
+  const networkDetails = network ?? NETWORKS.PUBLIC;
```

Comment:
could we please rename `networkDetails` to something else? If we look at `constants.ts` we'll see that `NetworkDetails` is an object type with several keys and here `networkDetails` is actually a `string` which is referring to a `NETWORKS` value.